### DISCUSSION TITLE
How do i open the App

### DISCUSSION THREAD

**Discussion by zxarde** [KadeArchive/Kade-Engine#904] [General]
I recently downloaded Kade Engine. I dont find the EXE. But it only shows me assets and something other. I searched in youtube. How do i Open fnf now?
![KadeDev-Kade-Engine-1 5 2-18-gd667e2e zip (Nicht lizenzierte Testversion) 2021-06-18 18_49_33 Uhr](https://user-images.githubusercontent.com/77019859/122593441-f7a51e80-d065-11eb-99cf-9be1cd781626.png)

**Comment by sonictheman06** [comment]
first extract that stuff into a folder then in that folder go to the top left and you should see "File" click that and then click "Open windows powershell" then when in powershell type "lime test windows" and let the game compile and then it should open

**Comment by puyoxyz** [comment]
https://github.com/KadeDev/Kade-Engine/issues/905#issuecomment-864244814
